cli/exp_mcp.go

```diff
@@ -106,12 +108,95 @@ func (*RootCmd) mcpConfigureClaudeDesktop() *serpent.Command {
 }
 
 func (*RootCmd) mcpConfigureClaudeCode() *serpent.Command {
+	var (
+		apiKey           string
+		claudeConfigPath string
+		projectDirectory string
+		systemPrompt     string
+		taskPrompt       string
+		testBinaryName   string
+	)
 	cmd := &serpent.Command{
 		Use:   "claude-code",
 		Short: "Configure the Claude Code server.",
```

Comment:
We should probably add in the help to specify the project dir, otherwise the MCP won't run and users may be confused.